psi4/src/psi4/psimrcc/operation_contraction.cc

```diff
@@ -331,511 +330,5 @@ void CCOperation::contract_in_core(double** A_matrix, double** B_matrix, double*
-    // Store the timing in moinfo
-    moinfo->add_dgemm_timing(timer.get());
-
-
-  }  // end of for loop over irreps
-
-  PartB_timing += PartB.get();
-  Timer PartC;
-  if(need_sort){
-    sort(T_left,T_right,T_matrix,1.0);
-    for(int irrep=0;irrep<moinfo->get_nirreps();irrep++)
-//       if(T_left->get_pairpi(h)*T_right->get_pairpi(h)>0)
-        delete[] T_matrix[h];
-    delete[] T_matrix;
-  }
-  PartC_timing += PartC.get();
-}
-
-void CCOperation::contract_in_core(double** A_matrix,double** B_matrix,double** C_matrix,int rows_A,int rows_B,int
-rows_C,int cols_A,int cols_B,int cols_C)
-{
-  // CASE A, worst case scenario
-  //
-  //            -----------------------------
-  //           |         ---------------     |
-  //           |        |               |    |
-  //   A[A_l][A_r] = B[B_i][B_c] 1@1 C[C_c][C_i]
-  //      |                  |
-  //       ------------------
-  //  Solution:
-  //  1) Transpose B,C
-  //  2) Perform contraction of the indices
-  //  3) Transpose the result and store in A ???
-  if(operation=="1@1"){
-    double beta = 1.0;
-    int m = rows_A;
-    int n = cols_A;
-    int k = rows_B;
-    if(m*n == 0) return;
-    else if (k != 0){
-      F_DGEMM("n","t",&n,&m,&k,&factor,&(C_matrix[0][0]),&n,&(B_matrix[0][0]),&m,&beta,&(A_matrix[0][0]),&n);
-    }
-  }
-  // CASE B
-  //
-  //            ------------------------
-  //           |         ---------------|----
-  //           |        |               |    |
-  //   A[A_l][A_r] = B[B_i][B_c] 1@2 C[C_c][C_i]
-  //      |                  |
-  //       ------------------
-  //  Solution:
-  //  1) Transpose B
-  //  2) Perform contraction of the indices ???
-  //  3) Transpose the result and store in A ??
-  if(operation=="1@2"){
-    double beta = 1.0;
-    int m = rows_A;
-    int n = rows_C;
-    int k = rows_B;
-    if(m*n == 0) return;
-    else if (k != 0){
-      F_DGEMM("t","t",&n,&m,&k,&factor,&(C_matrix[0][0]),&cols_C,&(B_matrix[0][0]),&m,&beta,&(A_matrix[0][0]),&cols_A);
-    }
-  }
-  // CASE C,
-  //
-  //       -------------       ---------
-  //      |             |     |         |
-  //   A[A_l][A_r] = B[B_i][B_c] 2@1 C[C_c][C_i]
-  //           |                             |
-  //            -----------------------------
-  //  Solution:
-  //  Perform contraction of the indices by calling BLAS with swapped C and B
-  if(operation=="2@1"){
-    int m = rows_B;
-    int n = cols_A;
-    int k = rows_C;
-    if(m*n == 0) return;
-    double beta = 1.0;
-    else if (k != 0){
-      F_DGEMM("n","n",&n,&m,&k,&factor,&(C_matrix[0][0]),&n,&(B_matrix[0][0]),&cols_B,&beta,&(A_matrix[0][0]),&n);
-    }
-  }
-  // CASE D, best case scenario
-  //
-  //       -------------       ---------------
-  //      |             |     |               |
-  //   A[A_l][A_r] = B[B_i][B_c] 2@2 C[C_c][C_i]
-  //           |                        |
-  //            ------------------------
```

Comment:
should any of these notes go into an attic repo instead of the ether?

Reply:
I see no reason for it. There are comparable notes for the actually-in-use `setup_contractions` function earlier in the file.